pymc3/distributions/continuous.py

```diff
@@ -26,6 +36,86 @@ class UnitContinuous(Continuous):
     def __init__(self, transform=transforms.logodds, *args, **kwargs):
         super(UnitContinuous, self).__init__(transform=transform, *args, **kwargs)
 
+def get_named_nodes(graph):
```

Comment:
My intuition is that it would be better to get this information from `model`. Is there a reason to do it this way?